pkgs/development/python-modules/lintrunner/default.nix

```
@@ -0,0 +1,55 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchFromGitHub,
+  maturin,
+  rustPlatform,
+  cargo,
+  rustc,
+  python,
+}:
+
+buildPythonPackage rec {
+  pname = "lintrunner";
+  version = "0.12.7";
+  format = "pyproject";
+
+  # NOTE: there is a PR open on the upstream repository for the Cargo.lock file.
+  # After that is merged we can change the owner and rev to the upstream
+  # repository instead of a fork.
+  src = fetchFromGitHub {
+    owner = "PrestonHager";
+    repo = pname;
```

```suggestion
    repo = "lintrunner;
```
`repo` and `pname` are semantically different.